CollAction/Dockerfile

```diff
@@ -1,7 +1,8 @@
-﻿FROM microsoft/aspnetcore:1.1.1
-ARG source
+﻿FROM microsoft/dotnet:1.1.2-sdk
```

Comment:
I think it's probably best if we use a *separate* docker image to develop, build and test. This is also the docker image used for the production server, it doesn't really need the sdk. Also, the aspnetcore probably has a few good tweaks in it for a production server.

Reply:
Fair point. What about a Dockerfile and a Dockerfile.local? The Dockerfile.local can run local dev and Dockerfile can remain where it is for prod.

Reply:
Dockerfile and Dockerfile.dev (or test)

Reply:
yep happy to do that. Will make the change.